src/RescriptEmbedLang.res

```diff
@@ -46,6 +46,14 @@ module Glob = {
   external glob: glob = "default"
 }
 
+module MicroMatch = {
+  type mm = {
+    makeRe: string => RegExp.t
+  }
+  @module("micromatch")
+  external mm: mm = "default"
+}
+
```

Comment:
`fast-glob` already depends on `micromatch`, so we can use this to test glob matches for free.